### ISSUE TITLE
Cannot use backup-utils to backup 3.0.0.rc1

### ISSUE THREAD

**Issue by weebl2000** [github/backup-utils#689]
```
weebl at tpw540 in ~/git/backup-utils on release-3.0.0.rc*
$ bin/ghe-backup 
Starting backup of github.onior.com with backup-utils v3.0.0.rc in snapshot 20210129T141403
backup-utils 3.0.0 or greater is required!

```

**Comment by sollie** [comment]
Had the same issue on restore.
Looks like a quick fix is to tweak `/usr/share/github-backup-utils/version`

**Comment by djdefi** [comment]
This has been fixed in GitHub Enterprise Server 3.0.0.rc2 --  https://docs.github.com/en/enterprise-server@3.0/admin/release-notes#3.0.0.rc2

You'll need to upgrade the appliance to rc2 to take advantage of this fix. 

If you continue to have issues, feel free to reach out to support via https://support.github.com/
